src/v2/BaseLoginRouter.js

```diff
@@ -25,15 +25,14 @@ import LanguageUtil from 'util/LanguageUtil';
 import AuthContainer from 'views/shared/AuthContainer';
 import Header from 'views/shared/Header';
 import AppState from './models/AppState';
-
+import sessionStorageHelper from './client/sessionStorageHelper';
 import {
   startLoginFlow,
   interactionCodeFlow,
-  clearTransactionMeta
+  configIdxJsClient,
```

Comment:
can we rename it to `idxClientConfig` instead of `idxClientMixin` ?

Reply:
I'm hesitate `idxClientMixin` is better name than `idxClientConfig`

Reply:
My sentence formation had typo.. I meant `idxClientConfig` or `idxClientMixin` instead of `configIdxJsClient`

Reply:
I think as method name, it usually starts with Verb?